Tools/python/ROOTTools.py

```diff
@@ -828,63 +846,55 @@ def data_mc_comparison(name,
                 for ibin in xrange(1, xax.GetNbins()+2):
                     first_binning.append(xax.GetBinLowEdge(ibin))
             else:
-                for ibin in xrange(1, xax.GetNbins()+2):
-                    if abs(first_binning[ibin] - xax.GetBinLowEdge(ibin)) > 1e-6:
-                        raise ValueError('inconsistent binning')
-            xax = None
-
-            if sample not in data_samples:
-                sample.hist.Scale(sample.partial_weight(sample._datamccomp_file) * int_lumi)
-                if int_lumi_bkg_scale is not None and sample not in signal_samples:
-                    sample.hist.Scale(int_lumi_bkg_scale)
-
-            move_overflows_into_visible_bins(sample.hist, move_overflows)
-
-            if rebin is not None:
-                sample.hist_before_rebin = sample.hist
-                rebin_name = sample.hist.GetName() + '_rebinned'
-                if type(rebin) in (list, tuple):
-                    rebin = array('d', rebin)
-                if type(rebin) == array:
-                    if rebin[-1] > sample.hist.GetXaxis().GetXmax():
-                        raise ValueError('rebin_last %f greater than axis max (ROOT will handle this arbitrarily)' % (rebin[-1], sample.hist.GetXaxis().GetXmax()))
-                    sample.hist = sample.hist.Rebin(len(rebin)-1, rebin_name, rebin)
```

Comment:
Did this no longer function as expected / intended? I am surprised to see it deleted.

Reply:
My mistake. Use UL*Bjet* one. Done.